packages/manager/modules/navbar/src/notifications-menu/template.html

```diff
@@ -8,15 +8,165 @@
+                <div
+                    class="hub-navbar-notification_header"
+                    data-ng-bind="menuLink.time"
+                ></div>
+                <!-- content -->
+                <div class="hub-navbar-notification_content">
+                    <!-- icon -->
+                    <span
+                        class="oui-icon oui-icon_bicolor oui-navbar-notification__icon"
+                        data-ng-class=":: {
+                            'oui-icon_bicolor': menuLink.level !== 'incident',
+                            'oui-icon-error_circle': menuLink.level === 'error',
+                            'oui-icon-success_circle': menuLink.level === 'success',
+                            'oui-icon-info_circle': menuLink.level === 'info',
+                            'oui-icon-warning_circle': menuLink.level === 'warning',
+                            'oui-icon-clock-wait': menuLink.level === 'incident'
+                        }"
+                        aria-hidden="true"
+                    >
+                    </span>
+
+                    <!-- Link (href) -->
+                    <a
+                        class="oui-navbar-notification__content"
+                        data-ng-if="::!!menuLink.url"
+                        data-ng-init="$ctrl.notificationsCreated = $last"
+                        data-ng-click="menuLink.linkClicked(menuLink);$ctrl.navbarCtrl.toggleMenu()"
+                        data-ng-href="{{::menuLink.url}}"
+                    >
+                        <div class="oui-navbar-notification__header">
+                            <span
+                                class="oui-navbar-notification__title"
+                                data-ng-bind=":: menuLink.subject"
+                            ></span>
+                            <span
+                                class="oui-status oui-status_error hub-status_error"
+                                data-ng-if="menuLink.isActive"
+                                data-translate="navbar_notification_new"
+                            >
+                            </span>
+                        </div>
+                        <div class="oui_navbar-notification__body">
+                            <div
+                                class="oui-navbar-notification__description"
+                                data-ng-bind-html=":: menuLink.description"
+                            ></div>
+                        </div>
+                    </a>
+                    <!-- /Link (href) -->
+
+                    <!-- Link (ui-sref) -->
+                    <a
+                        class="oui-navbar-notification__content"
+                        data-ng-if="::!!menuLink.state"
+                        data-ng-init="$ctrl.notificationsCreated = $last"
+                        data-ng-click="menuLink.linkClicked(menuLink);$ctrl.navbarCtrl.toggleMenu()"
+                        data-ui-sref="{{::$ctrl.getFullSref()}}"
```

Comment:
Do we need that part ? Can't we go without using `ui-sref` ?

Reply:
I don't know the history, why it was added. If API sends notification URL only in HREF, we can remove it. I need to look into the notification 2API code.